assets/js/admin/course-theme-sidebar.js

```diff
@@ -3,49 +3,152 @@
+	);
+
+	const globalLearningModeEnabled =
+		window.sensei?.senseiSettings?.sensei_learning_mode_all || false;
+
+	const firstLesson = useSelect( ( select ) => {
+		const { getBlocks, getBlockAttributes } = select( 'core/block-editor' );
+		const blocks = getBlocks();
+		const courseOutline = getFirstBlockByName(
+			courseOutlineBlockName,
+			blocks
+		);
+
+		if ( ! courseOutline ) {
+			return {};
+		}
+
+		for ( const moduleOrLesson of getBlocks( courseOutline.clientId ) ) {
+			if ( canPreview( moduleOrLesson ) ) {
+				return getBlockAttributes( moduleOrLesson.clientId );
+			}
+
+			if ( moduleOrLesson.name === courseModuleBlockName ) {
+				for ( const lesson of getBlocks( moduleOrLesson.clientId ) ) {
+					if ( canPreview( lesson ) ) {
+						return getBlockAttributes( lesson.clientId );
+					}
+				}
+			}
+		}
+
+		return {};
+	} );
+
+	let previewUrl = '';
+	if ( firstLesson?.id && currentPost?.id ) {
+		if (
+			globalLearningModeEnabled ||
+			currentPost.meta._course_theme === SENSEI_THEME
+		) {
+			previewUrl = `/?p=${ firstLesson.id }`;
+		} else {
+			previewUrl = `/?p=${ firstLesson.id }&learn=1&${ SENSEI_PREVIEW_QUERY }=${ currentPost.id }`;
+		}
+
+		if ( firstLesson.draft ) {
+			previewUrl = `${ previewUrl }&post_type=lesson`;
+		}
+	}
+
+	let customizerUrl = '';
+	if ( previewUrl ) {
+		customizerUrl = `/wp-admin/customize.php?url=${ encodeURIComponent(
+			previewUrl
+		) }`;
+	}
+
+	return { previewUrl, customizerUrl };
+};
 
 /**
  * Course Theme Sidebar component.
  */
 const CourseThemeSidebar = () => {
+	const globalLearningModeEnabled =
+		window.sensei?.senseiSettings?.sensei_learning_mode_all || false;
 	const [ theme, setTheme ] = useCourseMeta( '_course_theme' );
+	const { previewUrl, customizerUrl } = usePreviewAndCustomizerLinks();
 
 	return (
 		<PluginDocumentSettingPanel
 			name="sensei-course-theme"
-			title={ __( 'Course Theme', 'sensei-lms' ) }
+			title={ __( 'Course Styles', 'sensei-lms' ) }
 		>
-			<p>
-				{ __(
-					'This does not change the theme of your site. It only applies to logged in users when viewing the course.',
-					'sensei-lms'
-				) }
-			</p>
-			<SelectControl
-				label={ __( 'Theme', 'sensei-lms' ) }
-				hideLabelFromVision
-				value={ theme }
-				options={ [
-					{
-						label: __( 'Sensei Theme', 'sensei-lms' ),
-						value: SENSEI_THEME,
-					},
-					{
-						label: __( 'WordPress Theme', 'sensei-lms' ),
-						value: WORDPRESS_THEME,
-					},
-				] }
-				onChange={ setTheme }
-			/>
+			{ globalLearningModeEnabled ? (
+				<p className="sensei-course-theme-toggle__global-overwrite-notice">
+					<a href="/wp-admin/admin.php?page=sensei-settings#course-settings">
+						{ __(
+							'Learning Mode is enabled globally.',
+							'sensei-lms'
+						) }
+					</a>
+				</p>
+			) : (
+				<ToggleControl
+					label={ __( 'Learning mode', 'sensei-lms' ) }
+					help={
+						<>
+							<p className="sensei-course-theme-toggle__description">
```

Comment:
I was thinking that we could remove this `p` and the CSS styles for this. It will be a little different from the design, but it will match the WP standards, so it gets the same as other helper texts in the sidebar. Does it make sense?

Reply:
Absolutely! I removed that `p` element. 5f01634d45d0f59b5977e7e8560e13b0612d4a2f

But, let me know how do you want to render the preview link. It could either be part of the helper text or it could show up separately to make sure it's visible. Right now it's rendered separately like in the option 2.

| 1 | 2 |
|---|---|
|<img width="270" alt="Screen Shot 2022-01-11 at 12 25 07 PM" src="https://user-images.githubusercontent.com/2578542/148907363-dbdc626b-f20b-4c71-8c47-b840a1f49686.png">|<img width="272" alt="Screen Shot 2022-01-11 at 12 26 38 PM" src="https://user-images.githubusercontent.com/2578542/148907371-e644357c-370b-4c39-89fe-d2053f8fc56b.png">|